Gems/Atom/RPI/Code/Source/RPI.Public/Material/Material.cpp

```diff
@@ -598,33 +600,33 @@ namespace AZ
             }
             else
             {
-                if (!imageAsset.IsReady())
+                AZ::Data::AssetInfo assetInfo;
+                AZ::Data::AssetCatalogRequestBus::BroadcastResult(assetInfo, &AZ::Data::AssetCatalogRequests::GetAssetInfoById, imageAsset.GetId());
+
+                Data::Instance<Image> image = nullptr;
+                if (assetInfo.m_assetType == azrtti_typeid<StreamingImageAsset>())
                 {
-                    imageAsset = Data::AssetManager::Instance().GetAsset<StreamingImageAsset>(imageAsset.GetId(), AZ::Data::AssetLoadBehavior::PreLoad);
-                    imageAsset.BlockUntilLoadComplete();
-                    if (!imageAsset.IsReady())
-                    {
-                        AZ_Error(s_debugTraceName, false, "Image asset could not be loaded");
-                        return false;
-                    }
+                    Data::Asset<StreamingImageAsset> streamingImageAsset{imageAsset.GetId(), assetInfo.m_assetType};
```

Comment:
Wouldn't constructing a new Data<Asset> here cause it to lose reference?
```
Data::Asset<StreamingImageAsset> streamingImageAsset{imageAsset.GetId(), assetInfo.m_assetType};
```
This constructs a completely new Data::Asset<T> sets its id and type and refcount to 0 and data to null.  Is this ok?  It won't keep the asset loaded in memory if its already loaded since the asset manager's refcount of its instance won't be incremented.  Its true though that we probably don't want to automaticlaly load an image just because its referenced though.

Reply:
I updated to 
`Data::Asset<StreamingImageAsset> streamingImageAsset = Data::static_pointer_cast<StreamingImageAsset>(imageAsset);`